src/SpotifyAuthRedirect.jsx

```diff
@@ -5,7 +5,7 @@ import { setToken } from "./utils/tokenStorage";
 
 const getAccessTokenFromLocationHash = (locationHash) => {
   const parsed = queryString.parse(locationHash);
-  return parsed["access_token"];
+  return parsed["/access_token"];
```

Comment:
And here, I have to append '/', because callback url changed to {...}/#/access_token=... after using HashRouter